chain/rust/src/transaction/serialization.rs

```diff
@@ -2649,7 +2649,7 @@ impl Deserialize for TransactionBody {
                 update,
                 auxiliary_data_hash,
                 validity_interval_start,
-                mint,
+                mint: mint.map(Into::into),
```

Comment:
This is the one area we have to modify serialization code in exchange for being able to have an `AssetBundle` class. It's trivial to do so I don't think it's a big deal, and having our class like that (with Deref traits) lets you both treat it like it's still an OrderedHashMap in most contexts and also implement our own methods/traits over top of it.